source/extensions/filters/http/dynamic_modules/factory.cc

```diff
@@ -21,11 +21,13 @@ absl::StatusOr<Http::FilterFactoryCb> DynamicModuleConfigFactory::createFilterFa
                                       std::string(dynamic_module.status().message()));
   }
 
+  auto config_or_error = MessageUtil::anyToBytes(proto_config.filter_config());
```

Comment:
can you make this optional? currently, it's not required and would be great to retain that property as sometimes a module doesn't need configuration like

https://github.com/envoyproxy/dynamic-modules-examples/blob/d6d39109f573ba92e3c627adcf7b72e045878b98/integration/envoy.yaml#L82-L87